flags.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+func parseFlags() []string {
+	var (
+		printHelp    bool
+		printVersion bool
+		cfgFile      string
+		version      = "HEAD"
+	)
+
+	flagSet := pflag.NewFlagSet("d8-lint", pflag.ContinueOnError)
+
```

Comment:
`log-level` flag would be also nice here

Reply:
And how are we going to use it?

Reply:
DONE